### DISCUSSION TITLE
How to seed relational database with uuid

### DISCUSSION THREAD

**Discussion by mingzhe305** [prisma/prisma#15510] [Q&A] [answered]
Hello!

I am just trying to seed my database on Planetscale and followed the [tutorial](https://planetscale.com/blog/how-to-seed-a-database-with-prisma-and-next-js), but am unsure on the approach. Within my data.js file that I will be using to seed, how do I relate the uuid of topicID in topics model to topicID in questions model?

### **data.js (used for seeding):**
```
  const topics = [
    {
      topicName: 'P=V*I',
      topicLevel: 1,
    },
    {
      //default uuid of this topic to be passed into question
      topicName: "Ohm's Law",
      topicLevel: 2,
    },
  ];

  const questions = [
    {
      topicID; //uuid of Ohm's Law to be passed here
      questionContent: 'random content',
      questionDifficulty: 1,
    },
  ];
```

### **Topics model:**

```
model topics {
  topicID String @id @default(uuid())
  topicName String @db.VarChar(50)
  topicLevel Int
  topicID_rs questions[] @relation("topicID_rs")
}
```

### **Questions model:**
```
model questions {
  questionID String @id @default(uuid())
  topicID String
  questionContent String @db.VarChar(500)
  questionDifficulty Int //1-easy, 2-medium, 3-hard
  topicID_rs topics @relation("topicID_rs", fields: [topicID], references: [topicID])

  @@index([topicID])
}
```

**Comment by austincrim** [comment, marked as answer]
Hey @mingzhe305,

You can call `prisma.topic.create` first and use the returned value's `id` in the subsequent create queries.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
